### PR TITLE
DEV-2430: Force fee widget into separate row

### PR DESCRIPTION
**Pull request by nrh-cklimas**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

n/a

#### What's this PR do?

Fixes a cosmetic bug when there are an odd number of amount fields on a donation page.

I also added Storybook stories for this part of the donation page form to help future dev work.

#### Why are we doing this? How does it help us?

Improves the appearance of the donation page.

#### How should this be manually tested? Please include detailed step-by-step instructions.

1. Edit a donation page.
2. Edit the amounts so that there are an odd number of choices (e.g. 3 or 5).
3. Confirm that the "Agree to pay fees?" toggle is below the choices and is left-aligned with the choices.

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?

No.

#### Have automated unit tests been added? If not, why?

No--this is a visual change only and we're not testing appearance.

#### How should this change be communicated to end users?

n/a

#### Are there any smells or added technical debt to note?

No.

#### Has this been documented? If so, where?

No.

#### What are the relevant tickets? Add a link to any relevant ones.

https://news-revenue-hub.atlassian.net/browse/DEV-2430

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No.

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:

No.

### CODE REVIEW

File: spa/src/components/donationPage/pageContent/DAmount.styled.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ export const DAmount = styled.ul`
   }
 `;
 
+export const FeesContainer = styled.div`
+  grid-column: 1 / span 2;
```
Comment (nrh-cklimas) [inline spa/src/components/donationPage/pageContent/DAmount.styled.js:17]: This is the change that addresses the bug.

File: spa/src/components/donationPage/pageContent/DAmount.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { useState, useMemo } from 'react';
 import PropTypes from 'prop-types';
-import * as S from './DAmount.styled';
+import { DAmount as DAmountStyled, FeesContainer, FreqSubtext, OtherAmount, OtherAmountInput } from './DAmount.styled';
```
Comment (GuiMend) [inline spa/src/components/donationPage/pageContent/DAmount.js:3]: We could rename the original export style to `DAmountStyled` so you don't have to rename it here

**Review by GuiMend — APPROVED** [review]
LGTM 👍 
image

There is a weird marker on top of the story. Is it supposed to be like that?
image

**Comment by nrh-cklimas** [comment]
> There is a weird marker on top of the story. Is it supposed to be like that?

@GuiMend it's because the story is isolating out one part of the contribution page and it expects to be in a `<ul>` that styles  the list items a certain way--I didn't want to touch that aspect yet.

File: spa/src/components/donationPage/pageContent/DAmount.js (hunk 2 above)
Comment (nrh-cklimas) [reply spa/src/components/donationPage/pageContent/DAmount.js]: Good idea--did that in e4ca6428d4485cc51ebad99bf4c1b12e414303fe

**Review by x110dc — APPROVED** [review]

**Pull request merged**